doc/examples/filters/plot_deconvolution.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+=====================
+Image Deconvolution
+=====================
+In this example, we deconvolve an image using Richardson-Lucy
+deconvolution algorithm.
+
+The algorithm is based on Bayesian-based iterative method
+where an image blurred by a point-spread function is 
+recovered. 
+
+.. [1] William Hadley Richardson, "Bayesian-Based Iterative 
+       Method of Image Restoration",
+       J. Opt. Soc. Am. A 27, 1593-1607 (1972), doi:10.1364
+
+.. [2] http://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import color, data, restoration
+
+from scipy import ndimage as ndi
+
+astro = color.rgb2gray(data.astronaut())
+psf = np.ones((5, 5)) / 25
+astro_blurred = ndi.convolve(astro, psf)
+astro += 0.1 * astro.std() * np.random.standard_normal(astro.shape)
+
+deconvolved = restoration.richardson_lucy(astro, psf, 5)
+
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(8, 5), sharex=True, sharey=True)
+
+ax1.imshow(astro)
+ax1.set_title('Original image')
+ax1.axis('off')
+ax1.set_adjustable('box-forced')
+
+ax2.imshow(astro_blurred)
+ax2.set_title('Blurred Image')
+ax2.axis('off')
+ax2.set_adjustable('box-forced')
+
+ax3.imshow(deconvolved)
+ax3.set_title('Deconvolved using Richardson-Lucy')
+ax3.axis('off')
+ax3.set_adjustable('box-forced')
```

Comment:
We decided to move these configuration lines into a single `for` loop to reduce visual noise. See https://github.com/soupault/scikit-image/blob/7018540c9073593bc51f35760ad769ff1f329b81/doc/examples/xx_applications/plot_coins_segmentation.py#L43-L45.